interp/constrintern.ml

```diff
@@ -2052,19 +2052,17 @@ let empty_ltac_sign = {
   ltac_extra = Genintern.Store.empty;
 }
 
-let intern_gen kind env
+let intern_gen kind env sigma
                ?(impls=empty_internalization_env) ?(pattern_mode=false) ?(ltacvars=empty_ltac_sign)
                c =
```

Comment:
From the moment `intern_gen` knows the `sigma`, it should be possible to statically check that `CHole` nodes (representing a `?x`) refer to existing evars, correct?

Reply:
That's a question for @ppedrot but I think so.